pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MultiStageEngineCustomTenantIntegrationTest.java

```diff
@@ -139,13 +138,11 @@ protected Connection getPinotConnection() {
 
   @Override
   protected void overrideBrokerConf(PinotConfiguration brokerConf) {
-    brokerConf.setProperty(CommonConstants.Helix.CONFIG_OF_MULTI_STAGE_ENGINE_ENABLED, true);
     brokerConf.setProperty(QueryConfig.KEY_OF_QUERY_RUNNER_PORT, 8421);
```

Comment:
so we still need these port config? can we check whether it is possible to default dynamic port allocation?

Reply:
Those shouldn't matter, let me remove them as well

Reply:
The issue is instance config in helix is using port 0, so workers cannot talk to each other. Need to find the available port first then set it into server config.